### PR TITLE
added total youtuber count and currentPage in pagination

### LINKED ISSUE #5: Display number of total channels on the list.
Using js I think, we can display the number of total channels on the list.
under "Youtubers' Birthdays" above the table, put a text:

```
YouTubers on the list: <amount>
```

### PR DESCRIPTION
**Pull request by AshwanthramKL**
Fixes #5 
Changes made:
image

* Went ahead and displayed current page viewing in pagination div.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
 <body>
 	<div class="container">
 		<h1>YouTubers' Birthdays</h1>
+		<p>YouTubers on the list: <span class="count">0</span></p>
```
Comment (KendallDoesCoding) [inline index.html:10]: Can it be in the center of the page?

Comment (KendallDoesCoding) [inline index.html:10]: So, this will update automatically whenever I add a YouTuber, right?

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (AshwanthramKL) [reply index.html:10]: Yes it will

Comment (AshwanthramKL) [reply index.html:10]: Yes it can. I'll change it

Comment (AshwanthramKL) [reply index.html]: image

Check it out now

Comment (KendallDoesCoding) [reply index.html]: Perfect.

**Review by KendallDoesCoding — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
Thanks for your contribution @AshwanthramKL!!

**Pull request merged**